BUILD_VERSIONS

```diff
@@ -2,5 +2,5 @@
 export SYNAPSE_VERSION=v1.35.1
 export POSTGRES_VERSION=10
 export NGINX_VERSION=1.17
-export RAIDEN_VERSION=v2.0.0
+export RAIDEN_VERSION=v3.0.0rc8
```

Comment:
There is no docker image for that Raiden version on dockerhub. That's why your docker build CI steps are failing.
For some reason the Github workflow did not trigger the build of the `raidennetwork/raiden:v3.0.0rc8` image, while the newest PR was build under the `develop` tagged image

Reply:
But in any case, shouldn't you first release raiden 3.0.0 and then release the new RSB with using the `v3.0.0` raiden version here?

Reply:
Can one retrigger those builds?

We just set up the docker image build after the release. So in worst case I create a new RC

Reply:
Done